modules/kg-schemas/src/main/resources/schemas/nexus/core/distribution/v0.1.0.json

```diff
@@ -0,0 +1,62 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/nexus/core/quantitativevalue/v0.1.0",
+    "{{base}}/schemas/nexus/core/digest/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:DistributionShape",
+      "@type": "sh:NodeShape",
+      "and":[
+        {
+          "or": [
+            {
+              "path": "schema:downloadURL",
+              "name":"Download URL",
+              "nodeKind": "sh:IRI",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "schema:accessURL",
+              "name":"Access URL",
+              "nodeKind": "sh:IRI",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        },
+        {
+          "property": [
+            {
+              "path": "schema:contentSize",
+              "node": "{{base}}/schemas/nexus/core/quantitativevalue/v0.1.0/shapes/PositiveQuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nxv:digest",
```

Comment:
is `nxv` transitively defined?

Reply:
Yes. It is defined in the standards context:
{{base}}/contexts/nexus/core/standards/v0.1.0